### PR TITLE
Add blog post for meeting 24

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

File: freezeyt_blog/articles/meeting24.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+extra_pages:
+    - /extra.html
+extra_files:
+    /CNAME: "my.site.example"
+```
+
+Pro `extra_files` by bylo fajn umožnit vzít obsah ze souboru, ne jen z řetězce.
+Takové věci se dají docela jednoduše doplnit: zatím může být obsah zadaný jen
+jako řetězec (zde `"my.site.example"`, takže máme možnost místo toho použít
+slovník, který se dají specifikovat nejrůznější rozšíření.
```
Comment (adelpopelkova) [inline freezeyt_blog/articles/meeting24.md:72]:
```suggestion
slovník, kterým se dají specifikovat nejrůznější rozšíření.
```

File: freezeyt_blog/articles/meeting24.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+Pythonního balíčku s webovou aplikací.
+(Tohle dělá `frozen-flask`/`elsa` a pokud člověk nevyvíjí přímo aplikaci
+ale třeba jen obsah do ní, adresář `build` se objeví schovaný kdesi ve
+virtuálním prostředí.)
+Nejlepší tak bude při chybějícím `output` požadovat zadání adresáře na
+příkazové řádce, tak jako to děláme teď.
+
+Teď ale máme všechny tři varianty zamíchané do sebe.
+Kdyby někco začal přidávat další možnosti (např. kopírovat soubory přímo na
+vzdálený server, nebo použít jiné verzovací systémy než Git.), mohly by
```
Comment (adelpopelkova) [inline freezeyt_blog/articles/meeting24.md:128]:
```suggestion
vzdálený server, nebo použít jiné verzovací systémy než Git.), mohl by 
```

**Review by adelpopelkova — COMMENTED** [review]
Thanks, found a few typos. Otherwise looks good to me.

**Comment by encukou** [comment]
Díky za review!

**Pull request merged**